### ISSUE TITLE
Issue with boolean types

### ISSUE THREAD

**Issue by SnowyLeopard** [yosiat/panko_serializer#29]
Hello,

I started using your serializer gem and am running into a weird issue.

Whenever the model attribute is of type boolean i'm getting a '/panko_serializer-0.4.0/lib/panko/serializer.rb:92: [BUG] Segmentation fault at 0x00000000000003' error and the entire ruby/rails process crashes.

Whenever I remove the boolean attribute from the serializer it is working fine. Also when I use a custom method in the serializer like the following example it actually works (in the example 'admin' is a boolean):

```
class UserSerializer < Panko::Serializer
  attributes :id, :email, :name, :admin

  def admin
    object.admin
  end
end
```
So when I remove the admin method in the above example it produces the error.

Is this a known issue or am I doing something wrong?

I'm calling the serializer with:
```
UserSerializer.new.serialize_to_json(User.first)
```

More info:
Database; MySQL
Rails: 5.0.6
Ruby: 2.5.1

With kind regards,
Snowy Leopard

**Comment by yosiat** [comment]
@SnowyLeopard Found the bug see - https://github.com/yosiat/panko_serializer/commit/907b5da53f04b1414e6013465829ff8c64f3a3e4

Will release 0.4.1 version once CI is green

**Comment by SnowyLeopard** [comment]
Hey @yosiat!

Tested the github version in my app, can confirm this fixes the issue. Thanks!

regards,
Snowy Leopard

**Comment by yosiat** [comment]
@SnowyLeopard released 0.4.1 with a bug fix
